server/models/migrations/v0.8.0-v0.9.0/migrate.revised.sql

```diff
@@ -0,0 +1,248 @@
+/* #############################################################################
+  Update bhima database for the version 0.9.0
+ ############################################################################ */
+
+/**
+ * =============================================================================
+ * STRUCTURE
+ * =============================================================================
+*/
+
+-- Fix reference_uuid index bug
+ALTER TABLE `general_ledger` DROP INDEX `reference_uuid`;
+ALTER TABLE `posting_journal` DROP INDEX `reference_uuid`;
```

Comment:
I am pretty sure all our installation sites already have this properly set. I remember upgrading IMCK with this previously, but we never published it as a release :(.